dask_ml/model_selection/_incremental.py

```diff
@@ -757,7 +859,58 @@ def _get_params(self):
         else:
             return ParameterSampler(self.parameters, self.n_initial_parameters)
 
+    def _stop_on_plateau(self, info, instructions):
+        best_plateau = {}
+        out = {}
+        for k, steps in instructions.items():
+            records = info[k]
+            current_calls = records[-1]["partial_fit_calls"]
+            if self.max_iter and current_calls >= self.max_iter:
+                out[k] = 0
+            elif self.patience and current_calls >= self.patience:
+                plateau = [
+                    h["score"]
+                    for h in records
+                    if current_calls - h["partial_fit_calls"] <= self.patience
+                ]
+                if all(score <= plateau[0] + self.tol for score in plateau):
+                    out[k] = 0
+                    if len(best_plateau) == 0 or plateau[-1] >= max(
+                        best_plateau.values()
+                    ):
+                        best_plateau.update({k: plateau[-1]})
+                else:
+                    out[k] = steps
+
+            else:
+                out[k] = steps
+        return out, best_plateau
+
+    def fit(self, *args, **kwargs):
+        self._best_plateaus = {}
+        self._model_scores = {}
+        self._models_kept = []
```

Comment:
What are these values for?  Can you add a small comment to help other maintainers understand them in the future?